<?xml version="1.0"?>
<launch>

    <arg name="robot" default="$(find amr_ros)/config"/>

    <node pkg="amr_ros" name="auto_navi" type="auto_navi" args="$(arg robot)/auto_navi.conf" output="screen"/>    

</launch>
